### ISSUE TITLE
thanks for writing the post! it's extremely solid :D

### ISSUE THREAD

**Issue by jasikpark** [marcusbuffett/bevy_snake#9]
This isn't the right place, but I wanted to point out that your code blocks have their overflow hidden on the bevy tutorial post:

![image](https://user-images.githubusercontent.com/10626596/149281634-2fa303fd-cc17-40ed-9a89-e51699ae6841.png)

thanks for writing the post! I especially appreciate the "see the diff" asides that let me know whether I'm on track with your tutorial!

**Comment by marcusbuffett** [comment]
Thanks for the kind words! Especially about the “see the diff” bits, those are very annoying to update so I’m glad they’re appreciated 😄. 

The overflow thing is expected afaict, it’s scrollable right? I guess I could make the text wrap, but that looks a bit uglier IMO. What I can do is run these code blocks through rustfmt, with a small max_width, like 60 chars or something. I’ll do that, but lmk if that doesn’t sound right to you as a fix

**Comment by jasikpark** [comment]
Yeah, I'm a bit confused b/c it seems that scrolling isn't working for me - I eventually set "width:max-content" on the containing div... I'm not sure what's up with it.

Also I feel you on the "updating the diffs" standpoint b/c there was one or two diffs linked to in the blog post that didn't seem to contain the right changes

**Comment by jasikpark** [comment]
Maybe https://github.com/tummychow/git-absorb would be useful for you trying to edit git history easily?
